input/resources/valueset-au-jurisdiction-extended.xml

```diff
@@ -1,28 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ValueSet xmlns="http://hl7.org/fhir">
-    <id value="rsg-source-document-jurisdiction" />
+    <id value="au-jurisdiction-extended" />
     <meta>
         <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset" />
     </meta>
     <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
         <valueInteger value="0"/>
     </extension>
-    <url value="http://terminology.hl7.org.au/ValueSet/rsg-source-document-jurisdiction" />
+    <url value="http://terminology.hl7.org.au/ValueSet/au-jurisdiction-extended" />
```

Comment:
URL should be "http://terminology.hl7.org.au/ValueSet/jurisdiction-extended". 
Please see conventions for extending a core (or external) terminology in Australia: 
- convention for url: `<hl7 au namespace>/<external value set id>-extended` e.g. http://terminology.hl7.org.au/ValueSet/v3-ActEncounterCode-extended

Reply:
Yes, this would be my fault, there were errors thrown about this file's url and I have incorrectly attributed it to the id and url not matching.

Reply:
Thanks...We'll go with this canonical:
```
http://terminology.hl7.org.au/ValueSet/jurisdiction-extended
```

Reply:
I've fixed this

Reply:
Confirmed fix.